tests/util/helpers.py

```diff
@@ -192,18 +191,15 @@ def workspacesEqual(Workspace1: str, Workspace2: str, **other_options):
     Returns: if the workspaces are equal, will return True
     Otherwise, will raise an assertion error containing the result of CompareWorkspaces in description
     """
-    equal = False
-    if other_options == {}:
-        assert_wksp_almost_equal(Workspace1, Workspace2)
-        equal = True
-    else:
-        equal, message = CompareWorkspaces(
-            Workspace1=Workspace1,
-            Workspace2=Workspace2,
-            **other_options,
-        )
-        if not equal:
-            raise AssertionError(message.column("Message"))
+    # NOTE this can be re-worked to call `assert_wksp_almost_equal` when that
+    # has been fixed to allow exact comparisons.
+    equal, message = CompareWorkspaces(
+        Workspace1=Workspace1,
+        Workspace2=Workspace2,
+        **other_options,
+    )
+    if not equal:
+        raise AssertionError(message.column("Message"))
```

Comment:
Following on a discussion with @ekapadi, the former method of using mantid's `assert_almost_equal` does not allow exact comparisons, and many tests should be using exact comparisons (see `test_GroceryService` for example; cloned workspaces should not be almsot equal, but exactly equal).  If the `assert_almost_equal` method is fixed to allow exact comparison, then this could be restored to its former glory.  Probably another defect should re-replace all of the `assert_wksp_almost_equal` with this method or a properly-written `CompareWorkspaces`